rholang/src/main/scala/coop/rchain/rholang/interpreter/error_handling/package.scala

```diff
@@ -0,0 +1,25 @@
+package coop.rchain.rholang.interpreter
+
+import cats.implicits._
+import cats.MonadError
+import cats.effect.concurrent.Ref
+
+package object error_handling {
+
+  type _error[F[_]] = Ref[F, Option[Throwable]]
```

Comment:
To me it feels that error is not a good name. An "error" should be a value, not a mutable storage.

I would go with _errorHandler instead of _error. Or maybe _errorStore

Reply:
The current name is fine to me. It won't matter though for reasons pointed out in my future comment.